### PR TITLE
Feat/form management config

### PR DESCRIPTION
**Pull request by mrcuriosity-tw**
## Summary

Refactor config step page to make the page states controled by RHF.

## Before

_Description_

**Screenshots**
If applicable, add screenshots to help explain behavior of your code.

## After

_Description_

**Screenshots**
If applicable, add screenshots to help explain behavior of your code.

## Note

_Null_

### CODE REVIEW

**Review by PengxiWPix — APPROVED** [review]
LGTM

**Review by weiraneve — APPROVED** [review]

File: frontend/__tests__/containers/ConfigStep/PipelineTool.test.tsx [hunk 1]
```diff
@@ -24,7 +28,7 @@ import { rest } from 'msw';
 
 export const fillPipelineToolFieldsInformation = async () => {
   const tokenInput = within(screen.getByTestId('pipelineToolTextField')).getByLabelText(
-    'input Token',
+    'input token',
```
Comment (guzhongren) [inline frontend/__tests__/containers/ConfigStep/PipelineTool.test.tsx:31]: extract `'input token'` as a fixture

File: frontend/__tests__/containers/ConfigStep/PipelineTool.test.tsx [hunk 2]
```diff
@@ -224,4 +231,41 @@ describe('PipelineTool', () => {
...
+
+  it('should close alert modal when user manually close the alert', async () => {
+    setup();
+    await fillPipelineToolFieldsInformation();
+    const timeoutError = new TimeoutError('', AXIOS_REQUEST_ERROR_CODE.TIMEOUT);
+    pipelineToolClient.verify = jest.fn().mockImplementation(() => Promise.resolve(timeoutError));
+
+    await userEvent.click(screen.getByText(VERIFY));
+
+    expect(await screen.getByTestId('timeoutAlert')).toBeInTheDocument();
```
Comment (guzhongren) [inline frontend/__tests__/containers/ConfigStep/PipelineTool.test.tsx:243]: fixture

**Review by guzhongren — APPROVED** [review]

File: frontend/__tests__/containers/ConfigStep/PipelineTool.test.tsx (hunk 1 above)
Comment (mrcuriosity-tw) [reply frontend/__tests__/containers/ConfigStep/PipelineTool.test.tsx:31]: make sense, refactored.

**Pull request merged**